### ISSUE TITLE
Whitespace lost between interpolated expressions in cfoutput

### ISSUE THREAD

**Issue by sbleon** [jcberquist/commandbox-cfformat#117]
Input:
```
<cfcomponent>
  <cffunction name='foo'>
    <cfoutput>
      this #Now()# #Now()# collapses the space between them
    </cfoutput>
  </cffunction>
</cfcomponent>
```

Output:
```
<cfcomponent>
  <cffunction name='foo'>
    <cfoutput>
      this #Now()##Now()# collapses the space between them
    </cfoutput>
  </cffunction>
</cfcomponent>
```

We need the space between the two interpolated expressions to be preserved.

Config:
```
{
  "alignment.consecutive.assignments": false,
  "alignment.consecutive.params": false,
  "alignment.consecutive.properties": false,
  "array.empty_padding": false,
  "array.multiline.comma_dangle": false,
  "array.multiline.element_count": 1,
  "array.multiline.leading_comma": false,
  "array.multiline.leading_comma.padding": true,
  "array.multiline.min_length": 80,
  "array.padding": false,
  "binary_operators.padding": true,
  "brackets.padding": false,
  "comment.asterisks": "align",
  "for_loop_semicolons.padding": true,
  "function_anonymous.empty_padding": false,
  "function_anonymous.group_to_block_spacing": "spaced",
  "function_anonymous.multiline.comma_dangle": false,
  "function_anonymous.multiline.element_count": 1,
  "function_anonymous.multiline.leading_comma": false,
  "function_anonymous.multiline.leading_comma.padding": true,
  "function_anonymous.multiline.min_length": 80,
  "function_anonymous.padding": false,
  "function_anonymous.spacing_to_group": false,
  "function_call.casing.builtin": "pascal",
  "function_call.casing.userdefined": "camel",
  "function_call.empty_padding": false,
  "function_call.multiline.comma_dangle": false,
  "function_call.multiline.element_count": 1,
  "function_call.multiline.leading_comma": false,
  "function_call.multiline.leading_comma.padding": true,
  "function_call.multiline.min_length": 80,
  "function_call.padding": false,
  "function_declaration.empty_padding": false,
  "function_declaration.group_to_block_spacing": "spaced",
  "function_declaration.multiline.comma_dangle": false,
  "function_declaration.multiline.element_count": 1,
  "function_declaration.multiline.leading_comma": false,
  "function_declaration.multiline.leading_comma.padding": true,
  "function_declaration.multiline.min_length": 80,
  "function_declaration.padding": false,
  "function_declaration.spacing_to_group": false,
  "indent_size": 2,
  "keywords.block_to_keyword_spacing": "spaced",
  "keywords.empty_group_spacing": false,
  "keywords.group_to_block_spacing": "compact",
  "keywords.padding_inside_group": true,
  "keywords.spacing_to_block": "spaced",
  "keywords.spacing_to_group": false,
  "max_columns": 120,
  "metadata.multiline.element_count": 1,
  "metadata.multiline.min_length": 80,
  "method_call.chain.multiline": 3,
  "newline": "os",
  "parentheses.padding": false,
  "property.multiline.element_count": 1,
  "property.multiline.min_length": 80,
  "strings.attributes.quote": "single",
  "strings.convertNestedQuotes": false,
  "strings.quote": "single",
  "struct.empty_padding": false,
  "struct.multiline.comma_dangle": false,
  "struct.multiline.element_count": 1,
  "struct.multiline.leading_comma": false,
  "struct.multiline.leading_comma.padding": true,
  "struct.multiline.min_length": 80,
  "struct.padding": true,
  "struct.quote_keys": false,
  "struct.separator": ": ",
  "tab_indent": false,
  "tags.lowercase": true
}
```

**Comment by sbleon** [comment]
Workaround: put a line break in between the two interpolated expressions.

**Comment by jcberquist** [comment]
Thanks! This should be fixed in `v0.16.3`.

**Comment by sbleon** [comment]
Thank you!
